README.md

```diff
@@ -63,7 +63,7 @@ Learn more on how to [configure your project](https://docs.reactioncommerce.com/
 - [Reaction Commerce YouTube videos](https://www.youtube.com/user/reactioncommerce/videos)
 
 ## Join the community calls
-- [Reaction Community calls](http://getrxn.io/2rcCal): Join our biweekly community calls every other Wednesday at 7AM PST/10AM EST. 
+- [Reaction community calls](http://getrxn.io/2rcCal): Join our biweekly community calls every other Wednesday at 7AM PST/10AM EST. 
```

Comment:
Should we change this to reflect the rotating schedule of 7/9 am?

Reply:
@sophiehe @zenweasel who's making this change?

Reply:
I feel like this should be the PR creator? @sophiehe? Or we can just not do it at 9am any more/ever which I am also in favor of.

Reply:
`7AM PST/10AM EST/3PM GMT`